### ISSUE TITLE
Have a dedicated e-mail list for ACT feedback

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act-rules#110]
We're currently listing wai@w3.org, @shawna-slh suggested setting up a separate list for ACT so she doesn't have to forward things. I think we can use `public-wcag-act-comments@w3.org` for this, which we also used for feedback on the ACT Rules Format. The only drawback to that is that it is public, I'm not sure everyone who wants to raise issues will want to do it publicly, particularly when it comes to potentially reporting issues with implementation data.
